personality-insights/v3.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2017, 2020.
+ * (C) Copyright IBM Corp. 2020.
```

Comment:
Did we run both lint and fix copyrights here? there should be two dates

Reply:
I did run both but it appears the copyright linter missed this one. I'll fix it manually and make sure to check every file